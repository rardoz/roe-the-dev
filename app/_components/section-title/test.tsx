import { render } from '@testing-library/react'
import SectionTitle from '.'

describe('SectionTitle', () => {
  it('should render', () => {
    expect(
      render(<SectionTitle>Hello there!</SectionTitle>).container,
    ).toMatchSnapshot()
  })
})
