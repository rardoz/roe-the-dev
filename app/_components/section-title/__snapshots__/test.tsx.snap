// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle should render 1`] = `
<div>
  <div
    class="w-3/4 lg:w-1/2 mx-auto mt-8 -mb-4"
  >
    <h2
      class="text-center text-4xl font-bold text-purple-700"
    >
      Hello there!
    </h2>
    <hr
      class="mt-5 mb-16 border-dotted border-spacing-8 border-8 border-t-0 border-purple-300"
    />
  </div>
</div>
`;
